Development

BenjaminGarza opened the pull request.

BenjaminGarza committed:
Added basic layout and added README

MauricioRobayo committed:
Add main-content section

BenjaminGarza committed:
Added Nav bar, added font Literata

MauricioRobayo committed:
Add more sections

BenjaminGarza committed:
Added new section to bottom and added text

Also resized some elements

MauricioRobayo committed:
Add person-of-the-week section

BenjaminGarza committed:
Added and styled two more sections

MauricioRobayo committed:
Add bottom links and footer

MauricioRobayo committed:
Add screenshot to the README.md

Ispirett commented:
# Reviewing
Hey, folks @MauricioRobayo  @BenjaminGarza  great work with this project you are doing a fantastic job with this heat map :1st_place_medal: . There are just a few changes that Mircoverse requires before this project can be completed.


 ### Navigation bar
- [x] It should use the <nav> HTML tag
- [x] It should have seven separate containers (<div> is recommended)
- [x] Containers should be positioned using the `float` or `flex` display properties

### Main content

- [ ] The main content should be made from separate containers, stacked on top of each other and in the following arrangement:

* [x] All content should be wrapped with `<main>` HTML tags
* [x] Every child of main should be a `<section>` HTML tag pair
* [ ] Every child of section should be an `<article>` HTML tag pair

**Also great work with the read me**

### new gitflow guildelines
Also important note for feature HTML projects use a feature branch instead of development you can give that branch descriptive name that describes the feature you are implementing.


When finished with these changes please submit for another code review you can use this same pull request link.

BenjaminGarza committed:
Split Nav bar into seven seperate containers

BenjaminGarza committed:
Html Organization fixes

Put every child of main in a section tag and every child of section in an article tag

BenjaminGarza committed:
Organization fixes, README update

Moved header and footer and fixed a bug that caused an incorrect layout

BenjaminGarza committed:
Merge pull request #2 from BenjaminGarza/Organization

Organization

samgaco commented:
Hi @BenjaminGarza  and @MauricioRobayo  :vulcan_salute: 

Good job implementing the requested changes! Everything looks very cool and creative... **The project is approved! Congratulations** :tada: 

**You can merge to the master branch and submit the project completion form.** Regards! :muscle:
